packages/frontend/src/features/metricsCatalog/components/MetricsCatalogCategoryForm.tsx

```diff
@@ -219,51 +212,29 @@ export const MetricsCatalogCategoryForm: FC<Props> = memo(
         return (
             <Popover
                 opened={opened}
-                onChange={handlePopoverChange}
+                onClose={onClose}
                 position="bottom"
                 width={300}
                 withArrow
                 shadow="md"
                 withinPortal
             >
                 <Popover.Target>
-                    <Button
-                        variant="default"
-                        size="xs"
-                        compact
-                        pos="absolute"
-                        leftIcon={
-                            <MantineIcon
-                                color="gray.6"
-                                size={8}
-                                icon={IconPlus}
-                            />
-                        }
-                        fz={10}
-                        right={0}
-                        bottom={0}
-                        left="auto"
-                        styles={(theme) => ({
-                            leftIcon: {
-                                marginRight: 4,
-                            },
-                            root: {
-                                border: `dashed 1px ${theme.colors.gray[4]}`,
-                                visibility:
-                                    hovered || opened ? 'visible' : 'hidden',
-                            },
-                        })}
-                        onClick={() => setOpened((prev) => !prev)}
-                    >
-                        Add
-                    </Button>
+                    <UnstyledButton w="100%" pos="absolute" />
                 </Popover.Target>
-                <Popover.Dropdown p="xs">
+                <Popover.Dropdown
+                    p="xs"
```

Comment:
let's make this `p={0}`, then wrap the `TagInput` and `Text` below with a `Box`
`   <Box p="xs">`

and on the `Stack` below that has `Stack` and the `Button` as children, let's pass a `px='xs'`